web_pwa_oca/readme/ROADMAP.rst

```diff
@@ -1,5 +1,37 @@
-* Evaluate to extend ``FILES_TO_CACHE``
-* Evaluate to use a normal JS file for service worker and download data from a normal JSON controller
 * Integrate `Notification API <https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/showNotification>`_
 * Integrate `Web Share API <https://web.dev/web-share/>`_
 * Create ``portal_pwa`` module, intended to be used by front-end users (customers, suppliers...)
+* Current *John Resig's inheritance* implementation doesn't support ``async``
+  functions because ``this._super`` can't be called inside a promise. So we
+  need to use the following workaround:
+
+  - Natural 'async/await' example (This breaks "_super" call):
+
+    .. code-block:: javascript
+
+        var MyClass = OdooClass.extend({
+            myFunc: async function() {
+                const mydata = await ...do await stuff...
+                return mydata;
+            }
+        });
+
+  - Same code with the workaround:
+
+    .. code-block:: javascript
+
+        var MyClass = OdooClass.extend({
+            myFunc: function() {
+                return new Promise(async (resolve, reject) => {
+                    const mydata = await ...do await stuff...
+                    return resolve(mydata);
+                });
+            }
+        });
+
+* Fix issue when trying to run in localhost with several databases. The browser
+  doesn't send the cookie and web manifest returns 404.
```

Comment:
Did you try adding `server_wide_modules = web,web_pwa` to your `odoo.conf`?

Once that's done, you can make your controllers with `auth="none"` and infer the database from a parameter, similar to [what is done in mail_tracking](https://github.com/OCA/social/blob/da180c747df57c05bd1be0e504d0f1f13c94b36b/mail_tracking/controllers/main.py#L42-L52).

Well, this is not a requirement because IMHO the roadmap note is good enough for now.

Reply:
I can't solve this problem following your instructions :/